### ISSUE TITLE
5.5 Foul When a Light Comes On Before Calling "Question"

### ISSUE THREAD

**Issue by 24RMiller** [gryphonshafer/Quizzing-Rule-Book#187]
Section 5.5 describes a foul as:

- If any light goes on after the quizmaster calls "Question" and before they begin to discernibly read the question; discernibly includes the quizmaster's mouth forming a word shape
- During a reference or quote question, if any light goes on after the quizmaster calls "Question" and before they begin to discernibly read the chapter number

These statements, when read literally, imply that the light must come on after the quizmaster calls “Question.” Thus, if a light is on before the quizmaster called “Question,” it cannot be ruled as foul (unless it is considered an “infraction of the rules, in letter or spirit, as discerned by the quiz officials.”). 

I propose two potential solutions to clear up this problem:

1. Change the word “goes” to “is” in both of the above bullet points. They would then read: 
- If any light is on after the quizmaster calls "Question" and before they begin to discernibly read the question; discernibly includes the quizmaster's mouth forming a word shape
- During a reference or quote question, if any light is on after the quizmaster calls "Question" and before they begin to discernibly read the chapter number

This would then mean that quizzers would be responsible for their own lights, and if they can not turn their light off in time, it would be a foul.

2. Keep this section the same. Instead, a rule could be added under 3.2.1., 4.1.3., or somewhere else entirely that states:
“The quizmaster may not call “Question” until all lights are off. If a quizzer’s light remains on, he or she should inform the quizzer of this.”
This is a practice that most quizmasters already do, but by putting it in writing, it would take away any chance for inconsistency. For example, I have seen quizmasters warn one quizzer that their light is on, and then later in the quiz, not inform a different quizzer and give them a foul instead. 

As a quizzer myself, I am in favor of the second solution. I can say from first-hand experience that it is not always easy to hear my light when 11 other people on the platform are repeatedly clicking their seats the entire question introduction. Both solutions help fix this problem, but the second one is fairer and more forgiving for quizzers. However, if others have a different idea I would be curious to see it as well.

**Comment by jttower** [comment]
I agree that this is an issue that needs to be clarified.  I agree with @24RMiller that as written the rules imply that the light must come on after the QM lets go of the button, not that it remains on when they let go of the button.  Therefore I think a foul should only be called if the light goes on after the QM lets go of the button, not if their light was already on.  In addition, some benches have "tricky seats" that make it harder to hear the click.  Some QMs say it is a quizzer's responsibility to make sure their light is off, but often quizzers can't see their lights and can't hear their click over everyone else's.  I know that QMs have a lot to keep track of already and some don't want to have to make sure everyone's lights are off before they say "Question".  I also know that it can be annoying when quizzers want to click their lights all the way up to "Question".  It always makes me smile when I remember Lana Haight saying "Settle..." before beginning the question introduction or QM cadence.  This seemed like a good way to have the quizzers stop clicking their seats so there was never an issue of a light being on accidentally.  

I agree with @24RMiller about adding a bullet point under 3.2.1 Quizmasters and also under 4.1.3. Question Reading Structure. 
Something could also be added about throwing out and redoing the question if it has been determined that a quizzer's light remained on accidentally, but I don't think it is necessary.
